src/servers/ZoneServer2016/managers/worldobjectmanager.ts

```diff
@@ -1024,9 +1024,34 @@ export class WorldObjectManager {
             prop.lootItem(server, obj, 1, false);
           }
           break;
-        case 9347:
+        case ModelIds.TREASURE_CHEST:
           const rewardChest = server._lootableProps[a] as TreasureChest;
           if (rewardChest) rewardChest.triggerRewards(server);
+
+          const activeTimeout: { [spawnerId: number]: NodeJS.Timeout } = {};
```

Comment:
this would need to be held in the class like rn it just disapear in the void

Reply:
Got it👍🏼 just updated